battleship.py

```diff
@@ -199,7 +303,24 @@ def placeShip(data):
 Returns: None
 '''
 def clickUserBoard(data, row, col):
-    return
+    cordinates = [row,col]
+    ship = data["temp_ship"]
+    if data["num_ship"] == 5:
+       return
+    for i in range(len(ship)):
+        if ship[i] == cordinates:
+            return 
+    ship.append(cordinates)
+    data["temp_ship"] = ship
+    print(data["temp_ship"])

```

Comment:
You can directly work on data["temp_ship"], no need for a separate variable

Reply:
Also, resolve conflicts by deleting pycache and then clicking on the resolve conflicts button for battleship.py